### PR TITLE
Add CaptureSavePLY.hdev

### PR DESCRIPTION
**Pull request by SatjaSivcev**
This example shows how to acquire a 3D color point cloud from the camera and save it to a PLY file format.

### CODE REVIEW

**Comment by SatjaSivcev** [comment]
Reminder for review

**Comment by SatjaSivcev** [comment]
@chrisasc @torbsorb can you review this?

**Comment by SatjaSivcev** [comment]
I've made some changes. Please review again @torbsorb @chrisasc

I was wondering if we should actually combine this sample with Capture.hdev, CaptureHDR.hdev, etc. The thing is that for C++, C#, Python, all capture samples save to ZDF, which is not possible for Halcon (HDevelop). Then, maybe we should save to PLY by default.

What do you think?

File: source/Camera/Basic/CaptureSavePLY.hdev [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<c>* Switching to the single Frame mode</c>
+<l>set_framegrabber_param (AcqHandle, 'AcquisitionMode', 'SingleFrame')</l>
+<c></c>
+<c>* Configuring settings</c>
+<l>set_framegrabber_param (AcqHandle, 'Iris', 30)</l>
+<l>set_framegrabber_param (AcqHandle, 'ExposureTime', 10000)</l>
+<l>set_framegrabber_param (AcqHandle, 'Gain',1)</l>
+<l>set_framegrabber_param (AcqHandle, 'Brightness', 1)</l>
+<c></c>
+<c>* Grabing the image</c>
```
Comment (torbsorb) [inline source/Camera/Basic/CaptureSavePLY.hdev]: Grabbing

Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureSavePLY.hdev]: Done

**Comment by SatjaSivcev** [comment]
> 
> 
> I've made some changes. Please review again @torbsorb @chrisasc
> 
> I was wondering if we should actually combine the code from this sample with Capture.hdev, CaptureHDR.hdev, etc. (and remove visualization). The thing is that for C++, C#, Python, all capture samples save to ZDF, which is not possible for Halcon (HDevelop). Then, maybe we should save to PLY by default.
> 
> What do you think?

@torbsorb do you then agree with this suggestion?

**Comment by torbsorb** [comment]
> 
> 
> > I've made some changes. Please review again @torbsorb @chrisasc
> > I was wondering if we should actually combine the code from this sample with Capture.hdev, CaptureHDR.hdev, etc. (and remove visualization). The thing is that for C++, C#, Python, all capture samples save to ZDF, which is not possible for Halcon (HDevelop). Then, maybe we should save to PLY by default.
> > What do you think?
> 
> @torbsorb do you then agree with this suggestion?

Yes. Btw, with "we should save to PLY by default", you mean (only) with Halcon, right?

**Comment by SatjaSivcev** [comment]
Yes, only with HALCON.

**Pull request merged**